appConfig/appConfig.yaml

```diff
@@ -5,11 +5,11 @@
 - key: PAYMENT_TOPIC_ADDRESS
   value: "ffc-demo-payment"
 - key: PAYMENT_SUBSCRIPTION_ADDRESS
-  value: "payment"
+  value: "ffc-demo-payment"
```

Comment:
ffc-demo-payment-payment

Reply:
@rajeshkaremane : No, it is **namespace-topicsubscriptionname** . so above is right.

Reply:
Should be a separate ticket then